### DISCUSSION TITLE
[New Feature] Set Multi-Engineer on a per-building basis

### DISCUSSION THREAD

**Discussion by MNOrac** [Vinifera-Developers/Vinifera#1520] [Feature Requests]
### Description:
Multi-engineer is a useful balancing tool for protecting players from cheesy base rushes. It can also be a pain point if a mod introduces tech structures that are meant to be capturable and change owners easily.

### Possible Implementation:
Something along the lines of `IgnoreMultiEngineer= yes / no`, for the exact scenario outlined above. Possibly also something like `ForceMultiEngineer=yes` to allow for the inverse.

**Comment by Rampastring** [comment]
CnCNet ts-patches has a hack that causes Multi-Engineer to entirely ignore structures owned by the Neutral house: https://github.com/CnCNet/ts-patches/blob/master/src/mods/multi_engineer_ignore_neutral.asm

With the hack, pre-placed buildings owned by Neutral, whether base buildings or tech buildings, always get captured by a single engineer. Stealing those tech buildings from other players still takes multiple engineers or damaging the tech building though, so it might still be better to have the `IgnoreMultiEngineer` key for `BuildingTypes` (possibly alongside the functionality of this CnCNet hack).
